modules/back-end/src/Domain/SemanticPatch/Instruction.Models.cs

```diff
@@ -41,6 +41,11 @@ public class RuleConditionValues : TheRuleId
 }
 
 public class RuleVariations : TheRuleId
+{
+    public ICollection<RolloutVariation> RolloutVariations { get; set; }
+}
+
+public class DefaultRuleRolloutVariations
 {
     public ICollection<RolloutVariation> RolloutVariations { get; set; }
 }
```

Comment:
Is it necessary to create a `DefaultRuleRolloutVariations` class ?

Reply:
There is no ruleId here, so I don't want to reuse RuleVariations class